app/controllers/spree/admin/products_controller.rb

```diff
@@ -89,7 +89,7 @@ def clone
                             Spree.t('notice_messages.product_not_cloned')
                           end
 
-        redirect_to spree.edit_admin_product_url(@new)
+        redirect_to spree.admin_products_url
```

Comment:
Although this looks like a change in behaviour, I think the frontend JS on the old bulk edit screen was reloading the products list anyway. 👍 
We'll test the old screen just in case though.

Reply:
Yes, you are right. This is a slight change in behavior as compared to the old screen.
We have [another issue](https://github.com/openfoodfoundation/openfoodnetwork/issues/11737) which deals with actual behavior for the new screen.
This can be handled/updated in that issue I believe. 
Please let me know if you think otherwise. :)